src/main/scala/zio/kafka/admin/AdminClient.scala

```diff
@@ -585,6 +591,23 @@ object AdminClient extends Accessible[AdminClient] {
       ).unit
     }
 
+    /**
+     * Delete the specified consumer groups.
+     */
+    override def deleteConsumerGroups(
+      groupIds: Iterable[String],
+      options: Option[DeleteConsumerGroupsOptions]
+    ): Task[Unit] =
+      fromKafkaFutureVoid(
+        blocking.effectBlocking(
```

Comment:
Do we really need `blocking`? If I remember correctly @guizmaii we don't?

Reply:
Never received an answer: https://github.com/zio/zio-kafka/issues/492 :/

Reply:
Bad but simple argument: Almost all methods use `blocking`
Better argument: the `deleteConsumerGroups` processes driver requests and use synchronized blocks which may be blocking the thread.